fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -3716,20 +3717,36 @@ public ParseNode visitSimpleFunctionCall(StarRocksParser.SimpleFunctionCallConte
         String functionName = getQualifiedName(context.qualifiedName()).toString().toLowerCase();
 
         if (DATE_FUNCTIONS.contains(functionName)) {
-            if (context.expression().size() != 2) {
-                throw new ParsingException(
-                        functionName + " must as format " + functionName + "(date,INTERVAL expr unit)");
-            }
+            if (context.expression().size() == 2) {
+                Expr e1 = (Expr) visit(context.expression(0));
+                Expr e2 = (Expr) visit(context.expression(1));
+                if (!(e2 instanceof IntervalLiteral)) {
+                    e2 = new IntervalLiteral(e2, new UnitIdentifier("DAY"));
+                }
+                IntervalLiteral intervalLiteral = (IntervalLiteral) e2;
+
+                return new TimestampArithmeticExpr(functionName, e1, intervalLiteral.getValue(),
+                        intervalLiteral.getUnitIdentifier().getDescription(), "START");
+            } else if (context.expression().size() == 3 && functionName.equals(FunctionSet.TIME_SLICE)) {
```

Comment:
Time slice not belong to TimestampArithmeticExpr

Reply:
Have change to use TimeSliceTypeExpr.